example-jobs/envTestJob.groovy

```diff
@@ -2,7 +2,7 @@
 import static jenkins.automation.utils.EnvironmentUtils.isDev
 
 
-def env=${ENVIRONMENT}
+def env="${ENVIRONMENT}"
 if (isDev(env)){
```

Comment:
I guess I didn't see this before, but why do we pass in the environment to the isDev() method?

Can't isDev() check if ${ENVIRONMENT} == 'dev'?

If anything, I feel like the env variable should be encapsulated within the  class EnvironmentUtils { }

https://github.com/cfpb/jenkins-automation/blob/master/src/main/groovy/jenkins/automation/utils/EnvironmentUtils.groovy